pinax/blog/views.py

```diff
@@ -164,6 +183,7 @@ def blog_feed(request, section=None, feed_type=None):
     else:
         raise Http404()
 
+    # @@@ how do we handle scoped kwarg here
```

Comment:
So i think this is solved now by having access to the kwargs (i changed the method sig but just didn't update the code)